ResourceStatusSystem/Agent/ElementInspectorAgent.py

```diff
@@ -209,9 +209,10 @@ def _execute(self):
       except Queue.Empty:
         return S_OK()
 
-      self.log.verbose('%s ( %s / %s ) being processed' % (element['name'],
-                                                           element['status'],
-                                                           element['statusType']))
+      self.log.verbose('%s ( vO=%s / status=%s / statusType=%s ) being processed' % (element['name'],
```

Comment:
At least here put `VO` please :)

```suggestion
      self.log.verbose('%s ( VO=%s / status=%s / statusType=%s ) being processed' % (element['name'],
```